### DISCUSSION TITLE
How to pass route parameters for edit and create ?

### DISCUSSION THREAD

**Discussion by degodfather** [filamentphp/filament#8280] [Help] [answered]
How can I pass the 'station id' parameter when creating an episode in my filamentPHP application? I've made modifications to my EpisodeResource file like this:

```php
'edit' => Pages\EditEpisode::route(
    '/{station}/{record}/edit'
),
```

However, after these changes, I'm unable to perform update or create operations. It appears that the 'POST' request to '/livewire/update' is not taking the 'station id' into account. Should I consider switching to an HTTP query request instead?"

**Comment by leandrocfe** [comment, marked as answer]
```php
//EditEpisode.php
public $station = null;
```

then you can use `$this->station`
